CMSgov/design-system
# [WNMGDS-2530] Sketch plugin: Update swatches in place

## Summary

WNMGDS-2530

Building on https://github.com/CMSgov/design-system/pull/2729, I've figured out how to update all the swatches and preserve all the assignments to the old ones in the specific layer styles of the document. For instance, if we have a designed component that is referencing the `alert__background-color` for its fill style, that fill style will get updated when we switch themes with the plugin. Previously I believed the only way we could do that was searching through all the styles in the document and finding matches, but the code I have in this branch seems to work where we're just kind of updating the swatches in place and calling some API functions to update references.

Note that my previous method of replacing the document's whole `swatches` array seemed to be part of the problem.

- Theme tokens are now built into the plugin. Whatever tokens exist when the plugin is built, those tokens will be used in the various themes. Instead of selecting json files from your filesystem when you run the plugin, you select from a list of themes.
- Token values are updated "in place" so that references from layer styles are maintained
- Updated readme

## How to test

1. Pull down this branch and switch to it
2. Open a sketch document
3. Use the plugin and choose a theme
4. Create a thing that uses one of the token colors
5. Use the plugin again to change the theme
6. It should update the colors on the thing

## Checklist
